### ISSUE TITLE
npm "@antv/g6": "latest" - registerCombo undefined

### ISSUE THREAD

**Issue by giannisn01** [antvis/G6#1675]
in the npm repository the features related to "combo" do not seem available 

**TypeError
undefined is not a function (near '..._g2.default.registerCombo...')**

this is an example from https://g6.antv.vision/en/examples/item/customCombo#cRect

https://codesandbox.io/s/8tc0b

**Comment by K0I** [comment]
+1

**Comment by baizn** [comment]
@giannisn01  你提供的那个demo中把以下部分删掉后就没问题啊
```
const descriptionDiv = document.createElement('div');
descriptionDiv.innerHTML =
  'Click the right marker to collapse/expand the combo.';
const graphDiv = document.getElementById('container');
graphDiv.appendChild(descriptionDiv);
```

**Comment by 426-330** [comment]
G6.registerCombo is not a function

**Comment by Yanyan-Wang** [comment]
> G6.registerCombo is not a function

Hi, I did not see the error of `G6.registerCombo is not a function` in the demo.

Delete line 98-102
```
const descriptionDiv = document.createElement('div');
descriptionDiv.innerHTML =
  'Click the right marker to collapse/expand the combo.';
const graphDiv = document.getElementById('container');
graphDiv.appendChild(descriptionDiv);
```

**Comment by Yanyan-Wang** [comment]
Sorry, it is due to the building problem of v3.5.4. It is solved by v3.5.5 now.
